charts/ldap/Chart.yaml

```
@@ -3,4 +3,4 @@ description: A Helm chart for ldap.jenkins.io
 maintainers:
 - name: jenkins-infra-team@googlegroups.com
 name: ldap
-version: 3.1.1
+version: 3.1.2
```

```suggestion
version: 4.0.0
```

Major change as the base image is bumped to a new major version (LDAP 2.4.x -> 2.5.x, OpenSSL from 1.1.1 to 3.x, Debian from 10 to 12)